### DISCUSSION TITLE
why not playlists?

### DISCUSSION THREAD

**Discussion by ookla-ariel-ride** [meisnate12/Plex-Meta-Manager#112] [General]
Curious why collections would be used over playlists.  Any chance you might add the ability to create a playlists instead of a collection for a future release?

**Comment by meisnate12** [comment]
Personally, I just like how collections look better than playlists and I don't have to bother syncing them to each user. I can look into adding playlists in a future release.

**Reply by splitsec2** [reply]
Being able to do playlists could fulfill a similar but different purpose.  
For example a creating a playlist for watching 

- Star Wars, (https://editorial.rottentomatoes.com/guide/star-wars-movies-in-order/) or the 
- Chicago TV series franchise that has many cross overs between Fire, Med, Police, etc.... (https://petitcartonvert.tumblr.com/post/158289265736/chicago-franchise-episodes-timeline) or the 
- Marvel Universe (https://www.pocket-lint.com/tv/news/disney/147514-mcu-timeline-best-marvel-movie-show-viewing-order) which has TV and Movies mixed into one (and thus not possible with a collection).

Great work on the PMM regardless!

**Reply by meisnate12** [reply]
playlists have been added since this comment

**Comment by ookla-ariel-ride** [comment]
@meisnate12 maybe a screenshot of how this looks would be useful.  For me, there is a bit of effort to get this going and I am unsure what the end result looks like.  This is clearly a popular project with many happy users, so don't take my comment as a knock.

**Comment by SerinitySW** [comment]
> @meisnate12 maybe a screenshot of how this looks would be useful. For me, there is a bit of effort to get this going and I am unsure what the end result looks like. This is clearly a popular project with many happy users, so don't take my comment as a knock.

They look the same as any other plex collections. Here are some example pics for you though of my library:
https://i.imgur.com/0ACmOKH.jpg
https://i.imgur.com/oVtBmrQ.jpg

Of course one of the biggest advantages is that they're available to all in-home and remote users.

**Comment by nebakanezzer** [comment]
seems to be currently supported according to the main github install instructions, and the example referenced above is given (star wars), however, I cannot find any templates within the config examples
